### PR TITLE
Criteria refactorization

### PR DESCRIPTION
**Pull request by aburgosu**

### CODE REVIEW

File: src/main/java/com/fundation/logic/model/Criteria/Audio.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+ */
+package com.fundation.logic.model.Criteria;
+
+/**
+ * Implements the model class Audio and the getter and setter´s methods
+ *
+ * @authors Andres Burgos.
+ * @version 1.1
+ */
+
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/model/Criteria/Audio.java]: Please remove this line.

File: src/main/java/com/fundation/logic/model/Criteria/Image.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+ */
+package com.fundation.logic.model.Criteria;
+
+/**
+ * Implements the model class Image and the getter and setter´s methods
+ *
+ * @authors Andres Burgos.
+ * @version 1.1
+ */
+
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/model/Criteria/Image.java]: Please remove this line.

File: src/main/java/com/fundation/logic/model/Criteria/Multimedia.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+ */
+package com.fundation.logic.model.Criteria;
+
+/**
+ * Implements the model class Multimedia and the getter and setter´s methods
+ *
+ * @authors Andres Burgos.
+ * @version 1.1
+ */
+
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/model/Criteria/Multimedia.java]: Please remove this line.

File: src/main/java/com/fundation/logic/model/Criteria/Video.java [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+ */
+package com.fundation.logic.model.Criteria;
+
+/**
+ * Implements the model class Video and the getter and setter´s methods
+ *
+ * @authors Andres Burgos.
+ * @version 1.1
+ */
+
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/logic/model/Criteria/Video.java]: Please remove this line.

File: src/main/java/com/fundation/logic/model/Criteria/Common.java [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+package com.fundation.logic.model.Criteria;
+
+import java.util.Date;
+
+/**
+ * Implements the model class Common and the getter and setter´s methods
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/model/Criteria/Common.java]: Put a  point at last of the sentence

File: src/main/java/com/fundation/logic/model/Criteria/Criteria.java [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+  * This software is the confidential and proprietary information of Jalasoft.
+  * ("Confidential Information"). You shall not
+  * disclose such Confidential Information and shall use it only in
+  * accordance with the terms of the license agreement you entered into
+  * with Jalasoft.
+  */
+ package com.fundation.logic.model.Criteria;
+
+ /**
+  * Implements the model class Criteria and the getter and setter´s methods
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/model/Criteria/Criteria.java]: Put a  point at last of the sentence

File: src/main/java/com/fundation/logic/model/Criteria/Image.java [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+package com.fundation.logic.model.Criteria;
+
+/**
+ * Implements the model class Image and the getter and setter´s methods
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/model/Criteria/Image.java]: Put a  point at last of the sentence

File: src/main/java/com/fundation/logic/model/Criteria/Multimedia.java [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+package com.fundation.logic.model.Criteria;
+
+/**
+ * Implements the model class Multimedia and the getter and setter´s methods
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/model/Criteria/Multimedia.java]: Put a  point at last of the sentence

File: src/main/java/com/fundation/logic/model/Criteria/Video.java [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+package com.fundation.logic.model.Criteria;
+
+/**
+ * Implements the model class Video and the getter and setter´s methods
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/model/Criteria/Video.java]: Put a  point at last of the sentence

File: src/main/java/com/fundation/logic/model/Criteria/Audio.java [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+package com.fundation.logic.model.Criteria;
+
+/**
+ * Implements the model class Audio and the getter and setter´s methods
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/model/Criteria/Audio.java]: Put a  point at last of the sentence

**Pull request merged**